goreleaser not working

## Describe the bug

The CI build pipeline currently relies on goreleaser lates which at the time of writing is [v1.7.0](https://github.com/goreleaser/goreleaser/releases/tag/v1.7.0) and requires go 1.18.

Example of pipeline failing due to this bug: https://app.travis-ci.com/github/dikhan/terraform-provider-openapi/builds/249182084

## To Reproduce

Steps to reproduce the behavior:

1. Travis CI pipeline step runs: `go install github.com/goreleaser/goreleaser@latest` 
2. The code will fail with the following error:

````
# github.com/goreleaser/goreleaser/pkg/context
../../../../pkg/mod/github.com/goreleaser/goreleaser@v1.7.0/pkg/context/context.go:151:15: undefined: strings.Cut
note: module requires Go 1.18
````

## Expected behaviour

goreleaser is pinned to a previous version [v1.6.3](https://github.com/goreleaser/goreleaser/releases/tag/v1.6.3) that does not required Go 1.18 and the pipeline is able to release code just fine.

## Additional context
Add any other context about the problem here.

### Checklist (for admin only)

**Don't forget to go through the checklist to make sure the issue is created properly**:

- [x] I have added a corresponding label (bug) to the issue (right side menu)
- [x] I have added this issue to the 'API Terraform Provider' GitHub project (right side menu)

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,7 +13,7 @@ env:
   - TF_VERSION=0.13.6 PERFORM_DOCKER_LOGIN=true GO111MODULE=on
 
 install:
-  - go install github.com/goreleaser/goreleaser@latest
+  - go install github.com/goreleaser/goreleaser@v1.6.3
   - wget https://releases.hashicorp.com/terraform/"$TF_VERSION"/terraform_"$TF_VERSION"_linux_amd64.zip
   - unzip terraform_"$TF_VERSION"_linux_amd64.zip
   - sudo mv terraform /usr/local/bin/

